src/App.scss

```diff
@@ -1 +1,37 @@
-// styles go here
```

Comment:
Replace the line comment with a block comment for better consistency. Use /* styles go here */ instead of // styles go here.